src/game/shared/neo/neo_gamerules.h

```diff
@@ -208,7 +208,8 @@ class CNEORules : public CHL2MPRules, public CGameEventListener
 	int GetForcedSkin();
 	int GetForcedWeapon();
 	virtual const char* GetGameTypeName(void) OVERRIDE;
-	bool CanChangeTeamClassWeaponWhenAlive();
+	virtual const bool CanChangeTeamClassLoadoutWhenAlive();
+	virtual const bool CanRespawnAnyTime();
```

Comment:
Nit: The `const` is a bit redundant in the prototypes, because we return bool by value so it doesn't do anything.